backend/molgenis-emx2-webapi/src/main/java/org/molgenis/emx2/web/DownloadApiUtils.java

```diff
@@ -13,4 +14,8 @@ private DownloadApiUtils() {
   static boolean includeSystemColumns(Context ctx) {
     return String.valueOf(ctx.queryParam(INCLUDE_SYSTEM_COLUMNS)).equalsIgnoreCase("true");
   }
+
+  static boolean includeMembers(Context ctx) {
+    return String.valueOf(ctx.queryParam(INCLUDE_MEMBERS)).equalsIgnoreCase("true");
```

Comment:
this is another feature than the fix, and one that I don't think we should make.